test/libsolidity/syntaxTests/parsing/destructuring_assignment_no_variables.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function g() pure public returns (int, int, int) {}
+    function f() pure public {
+        // ensure this test is failing
```

Comment:
Please change this comment to tell what you explained in the pull request's description instead - we already see that it is supposed to fail because of the expectation below.